commands/config.go

```diff
@@ -525,3 +535,34 @@ func configSubpathCmdF(cmd *cobra.Command, _ []string) error {
 
 	return nil
 }
+
+// cloudRestricted checks if the config path is restricted to the cloud
+func cloudRestricted(cfg any, path []string) bool {
+	t := reflect.TypeOf(cfg)
+	v := reflect.ValueOf(cfg)
+
+	if t.Kind() == reflect.Ptr {
+		t = reflect.Indirect(v).Type()
+		v = reflect.Indirect(v)
+	}
+
+	if t.Kind() == reflect.Struct {
+		for i := 0; i < t.NumField(); i++ {
+			field := t.Field(i)
+			if len(path) == 0 || field.Name != path[0] {
+				continue
+			}
+
+			if field.Type.Kind() == reflect.Struct {
+				return cloudRestricted(v.Field(i).Interface(), path[1:])
+			}
+
+			accessTag := field.Tag.Get(model.ConfigAccessTagType)
+			if strings.Contains(accessTag, model.ConfigAccessTagCloudRestrictable) {
```

Comment:
Don't think there's a need to put the constant in the model package of the server. Just an internal constant should be fine.

Reply:
They are already defined in the `model` package :)

Reply:
Oh! Didn't realize it. And that's why I was wondering why isn't there any related server PR with this :P